### PR TITLE
👀  Show the returns API on production

### PR DESCRIPTION
**Pull request by yoan-myparcel**

### CODE REVIEW

File: specification/paths/Brokers.json [hunk 1]
```diff
@@ -120,7 +120,6 @@
                       "attributes": {
                         "required": [
                           "name",
-                          "domain",
```
Comment (yoan-myparcel) [inline specification/paths/Brokers.json:123]: This change is not part of this PR. Git decides to show it 🤷🏻‍♂️

File: specification/schemas/brokers/Broker.json [hunk 2]
```diff
@@ -15,9 +15,13 @@
               "example": "MyParcel.com"
             },
             "domain": {
-              "type": "string",
+              "type": [
+                "string",
+                "null"
+              ],
               "maxLength": 255,
-              "example": "app.sendmyparcel.com"
```
Comment (yoan-myparcel) [inline specification/schemas/brokers/Broker.json:20]: This change is not part of this PR. Git decides to show it 🤷🏻‍♂️

File: src/myparcelcom.js [hunk 3]
```diff
@@ -102,9 +102,6 @@
           'ShipmentSurcharges',
           'SystemMessages',
           'ExportLoginAttempts',
-          "Returns",
-          "ReturnMethods",
-          "ReturnReasons",
```
Comment (yoan-myparcel) [inline src/myparcelcom.js:107]: This is the only change part of this PR

File: src/myparcelcom.js [hunk 4]
```diff
@@ -102,9 +102,6 @@
...
           "ReturnReasonsManagement",
```
Comment (yoan-myparcel) [inline src/myparcelcom.js:105]: FYI I decided to keep the ReturnReasonsManagement hidden since it's not interesting for users.

**Review by M4tini — APPROVED** [review]

**Review by M4tini — APPROVED** [review]
